ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -28,8 +28,8 @@
  * @bug         No known bugs except for NYI items
  *
  * option1: Decoder mode of bounding box.
- *          Available: tflite-ssd (single shot multibox detector with priors.)
- *                     tf-ssd
+ *          Available: mobilenet-ssd (single shot multibox detector with priors.)
```

Comment:
plz consider to denote ssd model and version info for developers :)

Reply:
For the `Post Process` models,
I checked the `mobilenet_ssd_v1_coco_quant_postprocess.tflite` and `
mobilenet_ssd_v2_coco_quant_postprocess.tflite` have the same output dimension for each rank.

For the models NOT including the post process, however,
it is meaningless to denote the version of the model
because the output dimension is decided by converting the Tensorflow model to the TFLite model,
and the converter can decide the output tensor from the graph.